chpl/chpl-service/src/main/resources/errors.properties

```diff
@@ -45,6 +48,7 @@ listing.badIcsRelative=Could not process ICS relationship parent %s to child %s.
 listing.icsTrueAndNoParentsFound=The ICS value is 'true' which means this listing has inherited properties. It is required that at least one parent from which the listing inherits be provided.
 listing.icsFalseAndParentsFound=The ICS value is 'false' which means this listing does not have inherited properties. ICS parents must not be specified but at least one was found.
 listing.missingIcs=ICS is required.
+listing.ics00=ICS Code is listed as 00 so no parents may be specified from which the listing inherits.
```

Comment:
`ICS Code is 00, which means this Listing must not inherit from any other Listings